<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>admin &#8211; Biometrics by the Border</title>
	<atom:link href="http://biometric2017.org/author/admin/feed/" rel="self" type="application/rss+xml" />
	<link>http://biometric2017.org</link>
	<description>The International Biometric Society Australasian Region Conference, 26-30th November 2017</description>
	<lastBuildDate>Sun, 26 Nov 2017 23:12:52 +0000</lastBuildDate>
	<language>en</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.6</generator>

<image>
	<url>http://biometric2017.org/wp-content/uploads/2017/02/IBS-logo.png</url>
	<title>admin &#8211; Biometrics by the Border</title>
	<link>http://biometric2017.org</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>A Permutation Test For Comparing Predictive Values In Clinical Trials</title>
		<link>http://biometric2017.org/timetable/event/a-permutation-test-for-comparing-predictive-values-in-clinical-trials/</link>
		<pubDate>Sun, 19 Nov 2017 22:17:51 +0000</pubDate>
		<dc:creator><![CDATA[admin]]></dc:creator>
		
		<guid isPermaLink="false">http://biometric2017.org/?post_type=mp-event&#038;p=914</guid>
		<description><![CDATA[Screening tests or diagnostic tests are important for early detection and treatment of disease. There are four well-known measurements, sensitivity&#8230;]]></description>
				<content:encoded><![CDATA[<p>Screening tests or diagnostic tests are important for early detection and treatment of disease. There are four well-known measurements, sensitivity (SE), specificity (SP), positive predictive value (PPV) and negative predictive value (NPV) in diagnostic studies. For comparing SEs/SPs, McNemar test is widely used, but there are only few methods for the comparison of PPVs/NPVs. Moreover, all of these methods are based on large-sample theory.</p>
<p>So, in this talk, firstly, we investigate the performance of those methods when the sample size is small. In addition, we propose a permutation test for comparing two PPVs/NPVs we can apply even if the sample size is small. Finally, we show the performance of the proposed method with some existing methods via simulation studies.</p>
]]></content:encoded>
			</item>
		<item>
		<title>Robust Semiparametric Inference In Random Effects Models</title>
		<link>http://biometric2017.org/timetable/event/robust-semiparametric-inference-in-random-effects-models/</link>
		<pubDate>Sun, 12 Nov 2017 23:19:53 +0000</pubDate>
		<dc:creator><![CDATA[admin]]></dc:creator>
		
		<guid isPermaLink="false">http://biometric2017.org/?post_type=mp-event&#038;p=900</guid>
		<description><![CDATA[We report on recent work using semiparametric theory to derive procedures with desirable robustness and efficiency properties in the context&#8230;]]></description>
				<content:encoded><![CDATA[<p>We report on recent work using semiparametric theory to derive procedures with desirable robustness and efficiency properties in the context of inference concerning scale parameters for random effect models.</p>
]]></content:encoded>
			</item>
		<item>
		<title>Conference Closing Ceremony</title>
		<link>http://biometric2017.org/timetable/event/conference-close/</link>
		<pubDate>Sat, 04 Nov 2017 22:48:48 +0000</pubDate>
		<dc:creator><![CDATA[admin]]></dc:creator>
		
		<guid isPermaLink="false">http://biometric2017.org/?post_type=mp-event&#038;p=865</guid>
		<description><![CDATA[Closing Address by Louise Ryan Poster and Oral prize presentations]]></description>
				<content:encoded><![CDATA[<h6>Closing Address by Louise Ryan</h6>
<h6>Poster and Oral prize presentations</h6>
]]></content:encoded>
			</item>
		<item>
		<title>The Skillings-Mack Statistic For Ranks Data In Blocks</title>
		<link>http://biometric2017.org/timetable/event/the-skillings-mack-statistic-for-ranks-data-in-blocks/</link>
		<pubDate>Sat, 04 Nov 2017 22:47:36 +0000</pubDate>
		<dc:creator><![CDATA[admin]]></dc:creator>
		
		<guid isPermaLink="false">http://biometric2017.org/?post_type=mp-event&#038;p=864</guid>
		<description><![CDATA[Skillings and Mack gave a statistic for testing treatment differences for ranks data in blocks &#8211; both complete and incomplete&#8230;]]></description>
				<content:encoded><![CDATA[<p>Skillings and Mack gave a statistic for testing treatment differences for ranks data in blocks &#8211; both complete and incomplete blocks or blocks with missing values. This general statistic is thus quite useful. There is an R package for its calculation. However we illustrate a problem with tied ranks. Sensory evaluation data is used.</p>
]]></content:encoded>
			</item>
		<item>
		<title>Bias Correction In Estimating Proportions by Pooled Testing</title>
		<link>http://biometric2017.org/timetable/event/bias-correction-in-estimating-proportions-by-pooled-testing/</link>
		<pubDate>Sat, 04 Nov 2017 22:46:57 +0000</pubDate>
		<dc:creator><![CDATA[admin]]></dc:creator>
		
		<guid isPermaLink="false">http://biometric2017.org/?post_type=mp-event&#038;p=863</guid>
		<description><![CDATA[Pooled testing (or group testing) arises when units are pooled together and tested as a group for the presence of&#8230;]]></description>
				<content:encoded><![CDATA[<p>Pooled testing (or group testing) arises when units are pooled together and tested as a group for the presence of an attribute, such as a disease. We have encountered pooled testing problems in plant disease assessment and prevalence estimation of mosquito-borne viruses.</p>
<p>In the estimation of proportions by pooled testing, the MLE is biased, and several methods of correcting the bias have been presented in previous studies. We propose a new estimator based on the bias correction method introduced by Firth (1993), which uses a modification of the score function. Our proposed estimator is almost unbiased across a range of problems, and superior to existing methods. We show that for equal pool sizes the new estimator is equivalent to the estimator proposed by Burrows (1987), which has been used by many practitioners.</p>
]]></content:encoded>
			</item>
		<item>
		<title>The Performance Of Model Averaged Tail Area Confidence Intervals</title>
		<link>http://biometric2017.org/timetable/event/the-performance-of-model-averaged-tail-area-confidence-intervals/</link>
		<pubDate>Sat, 04 Nov 2017 22:46:23 +0000</pubDate>
		<dc:creator><![CDATA[admin]]></dc:creator>
		
		<guid isPermaLink="false">http://biometric2017.org/?post_type=mp-event&#038;p=862</guid>
		<description><![CDATA[Commonly in applied statistics, there is some uncertainty as to which explanatory variables should be included in the model. Frequentist&#8230;]]></description>
				<content:encoded><![CDATA[<p>Commonly in applied statistics, there is some uncertainty as to which explanatory variables should be included in the model. Frequentist model averaging has been proposed as a method for properly incorporating this &#8220;model uncertainty&#8221; into confidence interval construction. Such proposals have been of particular interest in environmental and ecological statistics.</p>
<p>The earliest approach to the construction of frequentist model averaged confidence intervals was to first construct a model averaged estimator of the parameter of interest consisting of a data-based weighted average of the estimators of this parameter under the various models considered. The model averaged confidence interval is centered on this estimator and has width proportional to an estimate of the standard deviation of this estimator. However, the distributional assumption on which this confidence interval is based has been shown to be completely incorrect in large samples.</p>
<p>An important conceptual advance was made by Fletcher &amp; Turek (2011) and Turek &amp; Fletcher (2012) who put forward the idea of using data-based weighted averages across the models considered of procedures for constructing confidence intervals. In this way the model averaged confidence interval is constructed in a single step, rather than first constructing a model averaged estimator.</p>
<p>We review the work of Kabaila et al (2016, 2017) which evaluates the performance of the model averaged tail area confidence interval of Turek &amp; Fletcher (2012) in the “test scenario” of two nested normal linear regression models. Our assessment of this confidence interval is that it performs quite well in this scenario, provided that the data-based weight function is carefully chosen.</p>
<p>References</p>
<p>Kabaila, P., Welsh, A.H., &amp; Abeysekera, W. (2016) Model-averaged confidence intervals. Scandinavian Journal of Statistics.</p>
<p>Kabaila, P., Welsh, A.H. and Mainzer, R. (2017) The performance of model averaged tail area confidence intervals. Communications in Statistics &#8211; Theory and Methods.</p>
]]></content:encoded>
			</item>
		<item>
		<title>The Parametric Cure Fraction Model of Ovarian Cancer</title>
		<link>http://biometric2017.org/timetable/event/the-parametric-cure-fraction-model-of-ovarian-cancer/</link>
		<pubDate>Sat, 04 Nov 2017 22:45:41 +0000</pubDate>
		<dc:creator><![CDATA[admin]]></dc:creator>
		
		<guid isPermaLink="false">http://biometric2017.org/?post_type=mp-event&#038;p=861</guid>
		<description><![CDATA[We propose incorporating the Gamma link function to the generalized Gamma using a mixture cure fraction model. The mathematical properties&#8230;]]></description>
				<content:encoded><![CDATA[<p>We propose incorporating the Gamma link function to the generalized Gamma using a mixture cure fraction model. The mathematical properties of the proposed model were explored and the inferences for the models were obtained. The proposed model called Gamma- generalized gamma mixture cure model (GGGMCM) will be validated using a real life data set on ovarian cancer.</p>
]]></content:encoded>
			</item>
		<item>
		<title>Deconstructing The Innate Immune Component Of A Molecular Network Of The Aging Frontal Cortex</title>
		<link>http://biometric2017.org/timetable/event/deconstructing-the-innate-immune-component-of-a-molecular-network-of-the-aging-frontal-cortex/</link>
		<pubDate>Sat, 04 Nov 2017 22:44:54 +0000</pubDate>
		<dc:creator><![CDATA[admin]]></dc:creator>
		
		<guid isPermaLink="false">http://biometric2017.org/?post_type=mp-event&#038;p=860</guid>
		<description><![CDATA[Alzheimer&#8217;s disease is pathologically characterized by the accumulation of neuritic Î²-amyloid plaques and neurofibrillary tangles in the brain and clinically&#8230;]]></description>
				<content:encoded><![CDATA[<p>Alzheimer&#8217;s disease is pathologically characterized by the accumulation of neuritic Î²-amyloid plaques and neurofibrillary tangles in the brain and clinically associated with a loss of cognitive function. The dysfunction of microglia cells has been proposed as one of the many cellular mechanisms that can lead to an increase in Alzheimer&#8217;s disease pathology. Investigating the molecular underpinnings of microglia function could help isolate the causes of dysfunction while also providing context for broader gene expression changes already observed in mRNA profiles of the human cortex.<br />
We have used mRNA sequencing to construct gene expression profiles of microglia purified from the cortex of 11 subjects from a longitudinal cohort of aging, Rush Memory Aging Project (MAP). By studying these microglia gene expression profiles in the context of tissue-level profiles of the cortex of 542 subjects from the MAP and Religious Orders Study (ROS) we address dual problems. By using information from the large ROSMAP cohort, we are able to isolate the genes which are strongly associated with immune response. Conversely, we illustrate that the microglia signature can be used to highlight predefined sets of coexpressed genes in ROSMAP that are highly enriched for microglia genes. Addressing these two questions allows us to identify sets of microglia specific genes which are associated with various Alzheimer&#8217;s disease traits further emphasizing the molecular consequences of microglia dysfunction in this disease. Specifically, we are able to identify a set of microglia related genes associated with tau and amyloid pathology as well as an activated microglial morphology.</p>
]]></content:encoded>
			</item>
		<item>
		<title>Genetic Analysis Of Renal Function In An Isolated Australian Indigenous Community</title>
		<link>http://biometric2017.org/timetable/event/genetic-analysis-of-renal-function-in-an-isolated-australian-indigenous-community/</link>
		<pubDate>Sat, 04 Nov 2017 22:44:11 +0000</pubDate>
		<dc:creator><![CDATA[admin]]></dc:creator>
		
		<guid isPermaLink="false">http://biometric2017.org/?post_type=mp-event&#038;p=859</guid>
		<description><![CDATA[In close consultation with the local land council, and with ethical approval from many ethics committees, we have performed a&#8230;]]></description>
				<content:encoded><![CDATA[<p>In close consultation with the local land council, and with ethical approval from many ethics committees, we have performed a genome-wide association study (GWAS) on a sample cohort from the 1990s.<br />
We also have a follow up data set of 120 study participants from 2014, with DNA sequence data.<br />
I will discuss the statistical analyses of these data sets, with respect to issues of degraded DNA, high error rates and correlated samples.</p>
]]></content:encoded>
			</item>
		<item>
		<title>Bayesian Regression with Functional Inequality Constraints</title>
		<link>http://biometric2017.org/timetable/event/bayesian-regression-with-functional-inequality-constraints/</link>
		<pubDate>Sat, 04 Nov 2017 22:43:24 +0000</pubDate>
		<dc:creator><![CDATA[admin]]></dc:creator>
		
		<guid isPermaLink="false">http://biometric2017.org/?post_type=mp-event&#038;p=858</guid>
		<description><![CDATA[We derive the exact boundary, in any direction, of functional (semi-infinite) multivariate inequality constraints on the parameter space of a&#8230;]]></description>
				<content:encoded><![CDATA[<p>We derive the exact boundary, in any direction, of functional (semi-infinite) multivariate inequality constraints on the parameter space of a regression model. A parameter space subject to multiple constraints is also possible. The calculated boundaries inform the use of an MCMC hit-and-run algorithm used to determine the posterior distribution of the constrained regression models. The method is applied to a forensic morphometric analysis of human skulls where monotonicity is required in more than one dimension which, to our knowledge, alternate methods are yet to achieve. Methods are compared to those currently available with one dimensional constraints and the results discussed.</p>
]]></content:encoded>
			</item>
	</channel>
</rss>
