<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
	<html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns:(null)0="http://www.w3.org/TR/REC-html40">


<head>

    <title>Hobart 2015 &#45; Invited Speakers</title>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <meta name="description" content="Biometrics by the Harbor" />
    <meta name="keywords" content="Hobart, Tasmania, Biometrics, Harbor, Harbour" /> 
    <link rel="stylesheet" type="text/css" media="screen" href="layout.css" />	<link rel="shortcut icon" href="images/favicon.ico" />

	<style type="text/css">
          h2 { color: #940202; } 
	  p.center {text-align: center; }
	</style>
    </head>
<body style="background-color: #FFE5B4"><a name="top">&nbsp;</a> 
<!-- color for the pictures on left -->
<div class="wrapper">

 
     <div class="headerMain">            <img src="images/Hobart/night_harbor_MONAligths_cr1_comp1.png" height="159px" width="734px"  alt="Biometrics by the Harbour, Hobart, Tasmania, November 30 - December 3, 2015"/>
    </div>

		   

<!-- MAIN BODY of PAGE--> <div class="bodyArea">
<!-- menu area -->        <div class="menuArea">
			<div class="menubutton"><a href="index.html">Home</a></div>
			<div class="menubutton"><a href="Invited speakers.html">Invited Speakers</a></div>
			<div class="menubutton"><a href="shortCourses.html">Short Courses</a></div>
			<div class="menubutton"><a href="scientific.html">Scientific Programme</a></div>
        		<div class="menubutton"><a href="social.html">Social Programme</a></div>
			<div class="menubutton"><a href="abstract.html">Abstract Submission</a></div>
			<div class="menubutton"><a href="registration.html">Registration</a></div>
			<div class="menubutton"><a href="travel.html">Travel</a></div>
			<div class="menubutton"><a href="accommodation.html">Accommodation</a></div>
			<div class="menubutton"><a href="awards.html">Awards</a></div>
			<div class="menubutton"><a href="organisers.html">Organisers</a></div>
			<div class="menubutton"><a href="sponsors.html">Sponsors</a></div>
        	<div class="menubutton" style="width: 160; height: 112">
                <img border="0" src="images/Hobart/boat_vertical_5815.JPG" width="151" height="230" align="middle" hspace="4">
                <img border="0" src="images/Hobart/sunset.JPG" width="151" height="120" align="middle" hspace="4">
                <img border="0" src="images/Hobart/investigator_boat.JPG" width="151" height="230" align="middle" hspace="4">
                <img border="0" src="images/Hobart/sail_boat.JPG" width="151" height="120" align="middle" hspace="4"></div>		
 			<div class="menubutton">&nbsp;</div>		
			<div class="buttonWebmaster"><a href="mailto:yuliya.karpievitch@utas.edu.au?subject=Biometrics%20website">Email the Webmaster</a></div>
         </div>


 

<!-- main content of page --> <div class="contentArea">
<a name="top-of-page" id="top-of-page"></a>

<h1 style="line-height: 100%; margin-top: 0; margin-bottom: 0">&nbsp;</h1> 

<h1 style="line-height: 100%; margin-top: 0; margin-bottom: 0">Invited Speakers</h1> 

<p>We are privileged to have a distinguished list of invited speakers that span the breadth of themes for the conference: Rosemary Bailey, Adrian Bowman, Richard Emsley, John Hinde, Thomas Lumley, Otso Ovaskainen, Jay Ver Hoef, and David Warton.

<!--<a href="Invited speakers.html#Ric_Coe">Rosemary Bailey</a>, 
<a href="Invited speakers.html#Noel_Cressie">Adrian Bowman</a>, 
<a href="Invited speakers.html#Ian_James">Richard Emsley</a>,
<a href="Invited speakers.html#Noel_Cressie">John Hinde</a>,
<a href="Invited speakers.html#Alan_Welsh">Thomas Lumley</a>, 
<a href="Invited speakers.html#Alan_Welsh">Otso Ovaskainen</a>,
<a href="Invited speakers.html#Christine_Mueller">Jay Ver Hoef</a>, and
<a href="Invited speakers.html#Alan_Welsh">David Warton</a>. -->
<!--XXX  and YYY  are also 
presenting separate <a href="http://www.biometrics.org.au/conferences/Mandurah2013/shortCourses.html"> short courses </a> on Sunday 1 December.</p> -->



<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4"><a href="http://www-circa.mcs.st-and.ac.uk/~rab/">Rosemary Bailey</a> </font></h3>
<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4">The design key in single- and multi-phase experiments </font></h3>
<p>Desmond Patterson introduced the design key in 1965 in the context of experiments on crop rotations.  It can be used whenever the treatments have factorial structure, the experimental units have a poset block structure, and an orthogonal design is required.  The design key gives an algorithm for allocating treatments to experimental units, and another algorithm for identifying which stratum contains which treatment effect.  These two properties make it a very useful tool when extended to multi-phase experiments. </p>
<p>
Bio: R. A. Bailey is Professor of Mathematics and Statistics at the University of St Andrews.  After a doctorate in finite group theory at the University of Oxford, she worked at the Open University for a few years.  She embraced statistics while working on restricted randomization and factorial designs as a post-doctoral research fellow at the University of Edinburgh.  She spent ten years at Rothamsted Experimental Station designing and analysing agricultural experiments before returning to academia in the University of London.  She moved to St Andrews in 2013.  She has also held visiting positions and fellowships in France, Australia, New Zealand, the USA and Brazil.
</p>



<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4"><a href="http://www.gla.ac.uk/schools/mathematicsstatistics/staff/adrianbowman/">Adrian Bowman</a> </font></h3>
<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4">Visualising the environment: data, models and graphics </font></h3>
<p>
This talk aims to reflect a little on the languages we use to explain, discuss and communicate statistical concepts, models and analysis, both within
our own community and beyond it.  There will be particular emphasis on how we communicate uncertainty.  Several types of analysis will be considered,
mostly involving flexible regression and focussed largely on different forms of spatiotemporal data.  However, there will be a strong focus on the
role of graphics, which can provide a powerful means of conceptual communication and give clear expressions of the insights provided by models, while
remaining true to the issues associated with uncertainty.
</p>
<p>
Bio: Adrian Bowman is a Professor of Statistics in the University of Glasgow.  He grew up in the seaside town of Prestwick in Scotland, followed by university education in Glasgow and Cambridge, in Mathematics and then Statistics.  Adrian's first academic job was in the University of Manchester but he subsequently moved to Glasgow, where most of his career has developed and where he is now Head of the School of Mathematics & Statistics.  He is the joint-author of a book on smoothing techniques and is very active in research, currently on environmental modelling, the analysis of anatomical shapes and of brain images.  Adrian also has a longstanding interest in educational technology, particularly in the role of graphics in aiding the understanding of statistical ideas.  He is married to Janet, who teaches maths, and they have three grown-up children.  A recent fun activity was paddling a sea kayak through the Corryvreckan (you may have to Google that) under strict supervision.
<p>Adrian is supported by the SSAI-WA branch's Frank Hansford-Miller fellowship during his visit.
<p>


<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4"><a href="http://www.manchester.ac.uk/research/richard.emsley">Richard Emsley</a> </font></h3>
<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4">Stratified medicine: the essential role of mechanisms evaluation </font></h3>
<p>The idea that a given treatment will have more benefit for some patients than for others is the underlying foundation of what is termed personalised or stratified medicine. If we can identify who these patients are before the decision to give a treatment is made, it will help prevent other patients being exposed to an unnecessary treatment that is likely to be of little or no benefit. As well as the obvious benefits to individual patients, it will also mean that the health care providers can save money on expensive treatment costs by targeting the right treatments to the right patients at the right time. In many clinical scenarios there are hypothesised to be multiple predictive markers that could be combined to target treatments, and often these are of different modalities such as clinical, genetic and imaging markers.  

In this talk, I will present the underlying concepts of stratified medicine in terms of benefits and harms and describe why the underlying treatment mechanism is fundamental to evaluation of targeted treatments. I will discuss the issue of choosing the appropriate scale of interaction (additive versus multiplicative), methods for evaluating and combining multiple predictive markers, and prospective trial designs which use analysis methods from the causal inference literature.
</p>
<p>
Bio: Richard is a Senior Lecturer in Biostatistics in the Centre for Biostatistics. He is also Visiting Lecturer in the Department of Biostatistics at the Institute of Psychiatry, Psychology and Neuroscience at King's College London. His research aims to answer three key questions: Are treatments effective? How do they work? Which groups are they most effective for? This involves the development of statistical methods for causal inference, efficacy and mechanisms evaluation and stratified medicine. He is the initiator and Chair of the steering group for the UK Causal Inference Meeting (UK-CIM).  He is involved in stratified medicine research programmes in schizophrenia, psoriasis, arthritis and cancer.
</p>



<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4"><a href=http://www.nuigalway.ie/maths/jh/>John Hinde</a> </font></h3>
<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4">A Medley of Mixtures </font></h3>
<p> Mixture models have now become an important tool in statistical modelling, with the EM algorithm being an important driver in their popularization and applicability. This talk will present the basic ideas and discuss a range of different mixture models and their use in addressing specific applied problems. Examples discussed will include: mixtures of non-linear regression models for fish growth studies; mixtures of smooth curves for clustering time-course microarray data; mixture orthogonal regression models for sugarcane SNP data; regression mixture models for outlier detection/accommodation. </p>


<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4"><a href=https://www.mcri.edu.au/users/dr-katherine-lee>Katherine Lee</a> </font></h3>
<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0">Multiple imputation: a miracle cure for missing data?<font size="4"></font></h3>
<p>
Researchers conducting clinical and epidemiological studies are inevitably faced with problems of drop-out and missing data. Importantly, the participants who drop out of a study are often those with poor health resulting in selection bias in the available data. Multiple imputation is gaining popularity as a strategy for handling missing data, since it enables all participants to be included in the analysis. However it is far from a panacea for dealing with missing data, and a number of important questions remain regarding its practical application.
</p>
<p>
In this talk I will provide a brief introduction to multiple imputation and will review some of the research surrounding this powerful and versatile approach for handling missing data conducted by our research group. This will include a discussion of when multiple imputation is likely to be beneficial over a complete case analysis, which imputation procedure to use, and the imputation of skewed, limited range and semi-continuous variables. Importantly I will highlight that multiple imputation is not a miracle cure and that it can introduce bias in the estimation of parameters of interest if the imputation model is not appropriate.
</p>
<p>
Bio: Katherine Lee is a senior biostatistician at the Murdoch Childrens Research Institute. She grew up in Bristol in the UK before moving to Nottingham where she obtained a Bachelor of Science in Mathematics from the University of Nottingham. Following this she obtained a Masters of Science in Medical Statistics from the University of Leicester UK and a PhD in Biostatistics from the University of Cambridge. Katherine spent the first 2 years of her working life at the Medical Research Council Clinical Trials Unit in London, before moving to Melbourne. Katherine has over ten years’ experience in the design, planning and analysis of randomised trials and observational studies, and is now Associate Director (Biostatistics) at the Melbourne Children's Trials Centre at the Royal Childrens Hospital. She is also an Honorary Fellow in the University of Melbourne. Her methodological interest is in multiple imputation for dealing with missing data.
</p>



<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4"><a href=https://www.stat.auckland.ac.nz/showperson?firstname=Thomas&surname=Lumley>Thomas Lumley</a> </font></h3>
<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4">Family-based genetic association modelling in a multistage sample </font></h3>
<p>The Hispanic Community Health Study/Study of Latinos (HCHS/SOL) is a US cohort study following a multistage probability sample of approximately 16000 Hispanics/Latinos. One component of HCHS/SOL is a set of large-scale genetic association studies for which the standard analysis would be linear mixed models with terms for ancestry and relatedness as well as shared environment. Currently, methodology is not available for incorporating complex sampling in mixed models except where the sampling structure  is nested in the model structure, which is not the case for HCHS/SOL.  I will talk about inferential aspects of the modelling, and about computational issues when dealing with hundreds of thousands of genetic variables and the complex sampling and mixed-model correlation structures. This is joint work with Xudong Huang and Alastair Scott. </p>

<p>
Bio: Thomas Lumley is Professor of Biostatistics at the University of Auckland. He has an undergraduate degree from Monash, an MSc from Oxford, and a PhD from the University of Washington, where he subsequently spent twelve years on the academic staff. His research interests include statistical computing, semiparametric statistics and its connections with survey sampling, cardiovascular epidemiology, and genomics.
</p>


<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4"><a href=https://tuhat.halvi.helsinki.fi/portal/en/persons/otso-ovaskainen%281db616b7-b1b7-42d8-9ff6-af08221bc82f%29.html>Otso Ovaskainen</a> </font></h3>
<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4">Analyzing community data with joint species distribution models: abundance, traits, phylogeny, co-occurrence and spatio-temporal structures </font></h3>
<p>A key aim in community ecology is to understand the factors that determine the identities and abundances of species found at any given locality. Central concepts in this research field include the regional and local species pools, environmental filtering and biotic assembly rules. Typical datasets involve a matrix of presence-absences (or abundances) for a group of species at different sites, some environmental and geographical characteristics of those sites, and possibly information on the ecological traits and phylogenetic relationships of the species. The analysis of such data have been traditionally based on ordination approaches, but there is increasing interest to move to model based approaches, in particular joint species distribution models. I present a joint species distribution model that captures the influences of environmental filtering at the community-level by measuring the amount of variation and covariation in the responses of individual species to various characteristics of their environment. The selection of the local species pool from the regional species pool involves both deterministic (e.g. systematic differences in dispersal abilities) and stochastic (e.g. spatio-temporal randomness in the realized distribution patterns) processes. Biotic assembly rules are reflected in the model with the help of an association matrix, which models positive or negative co-occurrence patterns not explained by the responses of the species to their environment. I use a latent factor approach to enable model parameterization with data on species-rich communities and thus with high-dimensional association matrices. I illustrate the performance of the approach both with simulated and real data. </p>

<p>
Bio: Otso Ovaskainen is an internationally renowned researcher at the interface between mathematics, statistics and biology. His work focusses on connecting general theories with empirical research, which has led to important contributions to metapopulation theory and dispersal theory. He has recently been interested in adapting modern statistical tools to ecology, including diffusion-based stochastic processes for the study of animal movement, and latent variable models for community ecology and the study of species interactions.

Otso is a Professor in the Department of Biosciences at the University of Helsinki, Finland as well as in the Norwegian University of Science and Technology in Trondheim, Norway. He has authored over 100 publications, including in Nature and Science. For his contributions to theoretical and population ecology, he won the prestigious 2009 Academy of Finland Award, awarded to only two scientists nationally each year.

</p>


<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4"><a href="https://sites.google.com/site/jayverhoef/">Jay Ver Hoef</a> </font></h3>
<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4"> Estimating Abundance from Counts in Large Data Sets of Irregularly Spaced Plots using Spatial Basis Functions  </font></h3>
<p> Monitoring plant and animal populations is an important goal for both academic research and management of natural resources. Successful management of populations often depends on obtaining estimates of their mean or total over a region. The basic problem considered in this paper is the estimation of a total from a sample of plots containing count data, but the plot placements are spatially irregular and non-randomized. Our application had counts from thousands of irregularly spaced aerial photo images. We used change-of-support methods to model counts in images as a realization of an inhomogeneous Poisson process that used spatial basis functions to model the spatial intensity surface. The method was very fast and took only a few seconds for thousands of images. The fitted intensity surface was integrated to provide an estimate from all unsampled areas, which is added to the observed counts. The proposed method also provides a finite area correction factor to variance estimation. The intensity surface from an inhomogeneous Poisson process tends to be too smooth for locally clustered points, typical of animal distributions, so we introduce several new overdispersion estimators due to poor performance of the classic one. We used simulated data to examine estimation bias and to investigate several variance estimators with overdispersion. A real example is given of harbor seal counts from aerial surveys in an Alaskan glacial fjord. 

<p>
Bio: Jay Ver Hoef a statistician for the National Marine Mammal Lab of the National Oceanic and Atmospheric Association (NOAA), U.S. Dept. of Commerce. He obtained a Ph.D. at Iowa State University in Statistics in 1991. Jay develops statistical methods and consults on a wide variety of topics related to marine mammals and stream networks. The Marine Mammal Lab is located in Seattle, Washington, although Jay lives in Fairbanks, Alaska.  His main statistical interests are in spatial statistics and Bayesian statistics, especially applied to ecological and environmental data.  Jay is a fellow of the American Statistical Association.
</p>



<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4"><a href="www.eco-stats.unsw.edu.au">David Warton </a> </font></h3>
<h3 style="line-height: 100%; margin-top: 0; margin-bottom: 0"><font size="4">The case of the missing model: the modernisation of multivariate analysis in ecology </font></h3>
<p> For the best part of four decades, multivariate analysis in ecology has diverged substantially from mainstream statistics, perhaps because state-of-the-art in 1980's statistics was not capable of handling the complexity frequently seen in multivariate abundance data simultaneously collected across many species. But the methods developed in the ecological literature, still widely used today, have some serious shortcomings that suggest they are fast approaching their use-by date. The statistical literature appears to be "catching up" with ecology, in part through technologies to fit quite flexible hierarchical models capable of accommodating key date structure.  There is a significant movement now to reunify multivariate analysis in ecology with modern statistical practices. Some key developments on this front will be reviewed, and immediate challenges identified.

<p>
Bio: David trained jointly between ecology and statistics to postgraduate level (at Sydney and Macquarie), and his research since then has been at the interface of these two disciplines, addressing what he considers to be a significant knowledge gap.  He has organised the Eco-Stats conference (this December at UNSW), and its previous iteration in 2013, to help create connections across these disciplines.  His research ranges from applied to theoretical - translating modern statistical tools to ecology, and developing new statistical methodology motivated by ecological problems, including contributions to high-dimensional data analysis, resampling, point process modelling and model selection.
 </p>


<img src="images/Hobart/all_sponsors.png" width="740" height="380" align="center"></a>         
